clientcontroller/opstackl2/consumer.go

```diff
@@ -486,3 +542,32 @@ func (cc *OPStackL2ConsumerController) Close() error {
 	cc.opl2Client.Close()
 	return cc.CwClient.Stop()
 }
+
+func (cc *OPStackL2ConsumerController) isDelegationActive(
+	btcDel *btcstakingtypes.BTCDelegationResponse,
+) (bool, error) {
+
+	btcstakingParams, err := cc.bbnClient.QueryClient.BTCStakingParams()
+	if err != nil {
+		return false, err
+	}
+
+	covQuorum := btcstakingParams.GetParams().CovenantQuorum
+	ud := btcDel.UndelegationResponse
```

Comment:
Since this never changes and the `isDelegationActive` function will be called upon every BTC delegation, how about we pass `btcstakingParams` as an argument to `isDelegationActive` rather than querying it everytime?